doc/source/examples.rst

```diff
@@ -284,57 +330,63 @@ ds_s#_an_#h_ha_nd  hands
 
 
 Count cues and outcomes
-=======================
+-----------------------
 We can now count the cues and outcomes in our event file using the
 
 
 pyndl.count module
-------------------
+^^^^^^^^^^^^^^^^^^
 and also generate id maps for cues and outcomes:
 
 .. code-block:: python
 
     >>> from pyndl import count
-    >>> cue_freq_map, outcome_freq_map = count.cues_outcomes(event_file_name='doc/data/levent.tab')
-    >>> cue_freq_map
-    >>> outcome_freq_map
+    >>> freq, cue_freq_map, outcome_freq_map = count.cues_outcomes(event_file_name='doc/data/levent.tab.gz')
+    >>> freq
+    12
+    >>> cue_freq_map  # doctest: +ELLIPSIS
+    Counter({...})
+    >>> outcome_freq_map  # doctest: +ELLIPSIS
+    Counter({...})
     >>> cues = list(cue_freq_map.keys())
     >>> cues.sort()
     >>> cue_id_map = {cue: ii for ii, cue in enumerate(cues)}
-    >>> cue_id_map
+    >>> cue_id_map  # doctest: +ELLIPSIS
+    {...}
     >>> outcomes = list(outcome_freq_map.keys())
     >>> outcomes.sort()
     >>> outcome_id_map = {outcome: nn for nn, outcome in enumerate(outcomes)}
-    >>> outcome_id_map
+    >>> outcome_id_map  # doctest: +ELLIPSIS
+    {...}
 
 
 Filter the events
-=================
+-----------------
 As we do not want to include the outcomes 'foot' and 'feet' in this example
 as well as their cues '#f', 'fo' 'oo', 'ot', 't#', 'fe', 'ee' 'et', we use the
 
 
 pyndl.preprocess module
------------------------
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Rendered font looks like heading, but according to the text it should be just emphasised